frontend/public/components/namespace.jsx

```diff
@@ -244,16 +269,26 @@ const ProjectTableRow = ({ obj: project, index, key, style, customData = {} }) =
         )}
       </TableData>
       <TableData className={projectColumnClasses[1]}>
+        <span className="co-break-word">
+          {project.metadata.annotations['openshift.io/display-name'] || (
```

Comment:
I'm not sure if all namespaces have `annotations`. Since it's omitted if empty, this could be a runtime error. I'd use `getDisplayName` which avoids the problem.

Reply:
Fixed.